agent/app/agent_capability.go

```diff
@@ -54,6 +55,7 @@ const (
 //    com.amazonaws.ecs.capability.ecr-auth
 //    com.amazonaws.ecs.capability.task-iam-role
 //    com.amazonaws.ecs.capability.task-iam-role-network-host
+//    com.amazonaws.ecs.capability.docker-volume-driver.${driverName}
```

Comment:
we've deprecated `com.*` style for attribute names. Can you please use the `ecs.docker-volume-driver.*` instead?

Reply:
Fixed, and added comment to not use capabilityPrefix for new capabilities.